web-ui/src/main/resources/catalog/components/viewer/geometry/GeometryService.js

```diff
@@ -309,7 +309,12 @@
                 '<geometry>' +
                 input + '</geometry></gml:feature></featureMembers>';
 
-            var feature = format.readFeatures(fullXml, {
+            var feature = format.readFeatures(
```

Comment:
I think this is due to the OL version used in GeoNetwork doesn't support GML 3.2.0? If so would be good to add a comment in the code, so if it's updated to a version that supports it, this code can be removed.

Reply:
I'm looking for funding to update OL version too ...